ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +73,25 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def check_mirroring_status(pattern_key="health", exp_pattern_value="OK"):
+    """
+    Validates current mirroring status matches with expected values
```

Comment:
It would be great if we have doc string for pattern_key and exp_pattern_value

Reply:
doc strings have been updated